extensions/arc/src/ui/dashboards/miaa/miaaDashboardOverviewPage.ts

```diff
@@ -206,8 +206,13 @@ export class MiaaDashboardOverviewPage extends DashboardPage {
 								cancellable: false
 							},
 							async (_progress, _token) => {
-								await this._controllerModel.azdataLogin();
-								return await this._azdataApi.azdata.arc.sql.mi.delete(this._miaaModel.info.name);
+								const loginSession = await this._controllerModel.acquireAzdataLoginSession();
```

Comment:
the acquireAzdataLoginSession should also be placed in the try block, right? it might also fail.

Reply:
Yeah but if it throws then the reason for the try block (disposing the session) is unnecessary.